tests/suite-integration/PluginFlyvemdmAgent.php

```diff
@@ -28,34 +28,46 @@
  * @link      https://flyve-mdm.com/
  * ------------------------------------------------------------------------------
  */
+
 namespace tests\units;
 
 use Glpi\Test\CommonTestCase;
 
-class PluginFlyvemdmAgent extends CommonTestCase {
+class PluginFlyvemdmAgent extends CommonTestCase
+{
 
    /**
     * @var string
     */
    private $minAndroidVersion = '2.0.0';
 
-   public function beforeTestMethod($method) {
+   public function setUp() {
       $this->resetState();
+   }
+
+   public function beforeTestMethod($method) {
       parent::beforeTestMethod($method);
       $this->setupGLPIFramework();
-      $this->boolean($this->login('glpi', 'glpi'))->isTrue();
```

Comment:
the test here is to ensure login succeeds. It is time consuming to find why a test fails due to the user not being logged. Please keep it.

Reply:
There is no point to test it if we are using the correct username/password values for login and this is not the main funcion of Agent test integration. For that case there should exist the unit or functional test in glpi core.

Reply:
Except when it fails for some reason :D . It happened to me, and wasted time before figuring out that the login failed.